browser/src/control/Control.ServerAuditDialog.ts

```diff
@@ -105,10 +105,57 @@ class ServerAuditDialog {
 		};
 	}
 
+	public isClipboardAvailable(
+		clipboard = navigator.clipboard,
+		isSecure = window.isSecureContext,
+	): boolean {
+		return !!clipboard && isSecure;
+	}
+
 	public open() {
 		const serverEntries = this.getEntries(app.serverAudit);
 		const clientEntries = this.getEntries(ClientAuditor.performClientAudit());
 
+		// allEntries = serveEntries + clientEntries
+		const allEntries = serverEntries.concat(clientEntries);
+
+		// ✅ Combine all audit entries into a single string for clipboard copying
+		function dedent(str: string) {
+			return str.replace(/^[ \t]+/gm, ''); // removes leading spaces/tabs on each line
+		}
+
+		const auditText =
+			allEntries
+				.map((entry) => {
+					const statusIcon = entry.columns[0]?.collapsed;
+					const statusText = entry.columns[1]?.text;
+					const helpText = entry.columns[2]?.text;
+					const helpLink = entry.columns[2]?.link;
+
+					const statusLabel =
+						statusIcon === 'serverauditok.svg'
+							? 'ok'
+							: statusIcon === 'serverauditerror.svg'
+								? 'error'
+								: 'unknown';
+
+					return dedent(`
+			======================================
+		
+			Status (${statusLabel}):
+			${statusText}
+		
+			Help (SDK: ${helpText}):
+			${helpLink}
+		`);
+				})
+				.join('') + '\n======================================\n';
+
+		const finalAuditText = `Summary of Server Audit Dialog:\n${auditText}`;
```

Comment:
We need to provide proper translation to other languages.
We use `_('string to translate')` to do that. But please provide there constant text only (no variable parameters).
So if there is some vairable, we replace it with token  `{1}` and then do   `translatedString.replace('{1}', variableValue)`.

So you need to split the text into pieces.